### PR TITLE
refactor(editor): Migrate `MainSidebar.vue` to composition API (no-changelog)

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
## Summary
As the PR title says ⬆️ 

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/MainSidebar.vue [hunk 1]
```diff
@@ -97,322 +381,6 @@
...
-			];
-			return items;
-		},
-		userIsTrialing(): boolean {
-			return this.cloudPlanStore.userIsTrialing;
-		},
-	},
-	async mounted() {
-		this.basePath = this.rootStore.baseUrl;
-		if (this.$refs.user) {
```
Comment (RicardoE105) [inline packages/editor-ui/src/components/MainSidebar.vue:306]: How are we referencing the user when the template does not have a ref=user property? 🤔

**Review by RicardoE105 — DISMISSED** [review]

**Review by RicardoE105 — DISMISSED** [review]

**Review by RicardoE105 — COMMENTED** [review]
to check why we are using that ref

File: packages/editor-ui/src/components/MainSidebar.vue (hunk 1 above)
Comment (MiloradFilipovic) [reply packages/editor-ui/src/components/MainSidebar.vue:207]: Good catch, looks like the `ref` is not there for a while. Let me check which element is expected here and I'll update

**Review by RicardoE105 — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.58.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.58.0)

**Pull request merged**